09/03/2018 02:14:05.0924	[raft_python_callback.py write_to_logger 74]  924 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:init_state_functions_handler, LINE:28
MESSAGE:start
-----------------------------

09/03/2018 02:14:05.0924	[raft_python_callback.py write_to_logger 74]  924 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:init_state_functions_handler, LINE:70
MESSAGE:end
-----------------------------

09/03/2018 02:14:05.0924	[raft_python_callback.py write_to_logger 74]  924 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:init_multicast_message, LINE:81
MESSAGE:init multicast socket
-----------------------------

09/03/2018 02:14:05.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:init_multicast_message, LINE:88
MESSAGE:multicast was opened successfully
-----------------------------

09/03/2018 02:14:05.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/static_queue.c, FUNCTION:init_raft_queue, LINE:13
MESSAGE:first time
-----------------------------

09/03/2018 02:14:05.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:create_alarm_timer, LINE:42
MESSAGE:create new alarm timer
VARIABLES:  sharedRaftData.raft_state.current_state=0  timeout=2804 
-----------------------------

09/03/2018 02:14:05.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=leader_id 
-----------------------------

09/03/2018 02:14:05.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=0 
-----------------------------

09/03/2018 02:14:05.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=commit_idx 
-----------------------------

09/03/2018 02:14:05.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=0 
-----------------------------

09/03/2018 02:14:05.0930	[raft_python_callback.py write_to_logger 74]  930 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=applied_last_idx 
-----------------------------

09/03/2018 02:14:05.0930	[raft_python_callback.py write_to_logger 74]  930 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=0 
-----------------------------

09/03/2018 02:14:05.0932	[raft_python_callback.py write_to_logger 74]  932 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=status 
-----------------------------

09/03/2018 02:14:05.0932	[raft_python_callback.py write_to_logger 74]  932 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=-3 
-----------------------------

09/03/2018 02:14:05.0933	[raft_python_callback.py write_to_logger 74]  933 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=term 
-----------------------------

09/03/2018 02:14:05.0933	[raft_python_callback.py write_to_logger 74]  933 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=0 
-----------------------------

09/03/2018 02:14:05.0935	[raft_python_callback.py write_to_logger 74]  935 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:init_raft, LINE:152
MESSAGE:init succeed
-----------------------------

09/03/2018 02:14:05.0935	[raft_python_callback.py write_to_logger 74]  935 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_raft, LINE:192
MESSAGE:start raft...
VARIABLES:  raft_ip=230.1.1.10 
-----------------------------

09/03/2018 02:14:05.0935	[raft_python_callback.py write_to_logger 74]  935 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_raft, LINE:194
MESSAGE:start raft...
VARIABLES:  raft_port=12345  server_id=1  members_num=2  leader_timeout=500 
-----------------------------

09/03/2018 02:14:05.0936	[raft_python_callback.py write_to_logger 74]  936 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:05.0936	[raft_python_callback.py write_to_logger 74]  936 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_raft, LINE:199
MESSAGE:run raft...
-----------------------------

09/03/2018 02:14:07.0495	[raft_python_callback.py write_to_logger 74]  495 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=20 
-----------------------------

09/03/2018 02:14:07.0495	[raft_python_callback.py write_to_logger 74]  495 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:07.0495	[raft_python_callback.py write_to_logger 74]  495 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=8  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:07.0495	[raft_python_callback.py write_to_logger 74]  495 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:07.0495	[raft_python_callback.py write_to_logger 74]  495 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=8  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:07.0496	[raft_python_callback.py write_to_logger 74]  496 root [DEBUG] 
FILE:./C/Code/follower_handlers.c, FUNCTION:follower_vote_req_handler, LINE:153
MESSAGE:follower get vote request msg
VARIABLES:  sharedRaftData.raft_state.term=0  sharedRaftData.raft_state.did_I_vote=0  node->term=11  node->message_sent_by=2 
-----------------------------

09/03/2018 02:14:07.0496	[raft_python_callback.py write_to_logger 74]  496 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=term 
-----------------------------

09/03/2018 02:14:07.0496	[raft_python_callback.py write_to_logger 74]  496 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=11 
-----------------------------

09/03/2018 02:14:07.0498	[raft_python_callback.py write_to_logger 74]  498 root [DEBUG] 
FILE:./C/Code/static_queue.c, FUNCTION:create_new_queue_node_data, LINE:107
MESSAGE:create new msg
VARIABLES:  event=7 
-----------------------------

09/03/2018 02:14:07.0498	[raft_python_callback.py write_to_logger 74]  498 root [DEBUG] 
FILE:./C/Code/follower_handlers.c, FUNCTION:follower_vote_req_handler, LINE:171
MESSAGE:follower sending vote msg
-----------------------------

09/03/2018 02:14:07.0498	[raft_python_callback.py write_to_logger 74]  498 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:send_raft_message, LINE:100
MESSAGE:try to send msg
VARIABLES:  message_size=20 
-----------------------------

09/03/2018 02:14:07.0498	[raft_python_callback.py write_to_logger 74]  498 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:send_raft_message, LINE:118
MESSAGE:msg was successfully sent
VARIABLES:  message_size=20 
-----------------------------

09/03/2018 02:14:07.0498	[raft_python_callback.py write_to_logger 74]  498 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=20 
-----------------------------

09/03/2018 02:14:07.0498	[raft_python_callback.py write_to_logger 74]  498 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=1  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:07.0498	[raft_python_callback.py write_to_logger 74]  498 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:07.0716	[raft_python_callback.py write_to_logger 74]  716 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:07.0716	[raft_python_callback.py write_to_logger 74]  716 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:07.0716	[raft_python_callback.py write_to_logger 74]  716 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:07.0717	[raft_python_callback.py write_to_logger 74]  717 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:07.0717	[raft_python_callback.py write_to_logger 74]  717 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:07.0717	[raft_python_callback.py write_to_logger 74]  717 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=leader_id 
-----------------------------

09/03/2018 02:14:07.0717	[raft_python_callback.py write_to_logger 74]  717 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=2 
-----------------------------

09/03/2018 02:14:07.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:07.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:08.0212	[raft_python_callback.py write_to_logger 74]  212 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:08.0212	[raft_python_callback.py write_to_logger 74]  212 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:08.0212	[raft_python_callback.py write_to_logger 74]  212 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:08.0212	[raft_python_callback.py write_to_logger 74]  212 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:08.0212	[raft_python_callback.py write_to_logger 74]  212 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:08.0624	[raft_python_callback.py write_to_logger 74]  624 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:08.0624	[raft_python_callback.py write_to_logger 74]  624 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:08.0624	[raft_python_callback.py write_to_logger 74]  624 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:08.0624	[raft_python_callback.py write_to_logger 74]  624 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:08.0624	[raft_python_callback.py write_to_logger 74]  624 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:09.0134	[raft_python_callback.py write_to_logger 74]  134 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:09.0134	[raft_python_callback.py write_to_logger 74]  134 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:09.0134	[raft_python_callback.py write_to_logger 74]  134 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:09.0134	[raft_python_callback.py write_to_logger 74]  134 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:09.0135	[raft_python_callback.py write_to_logger 74]  135 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:09.0645	[raft_python_callback.py write_to_logger 74]  645 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:09.0645	[raft_python_callback.py write_to_logger 74]  645 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:09.0645	[raft_python_callback.py write_to_logger 74]  645 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:09.0645	[raft_python_callback.py write_to_logger 74]  645 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:09.0646	[raft_python_callback.py write_to_logger 74]  646 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:09.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:09.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:10.0157	[raft_python_callback.py write_to_logger 74]  157 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:10.0157	[raft_python_callback.py write_to_logger 74]  157 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:10.0157	[raft_python_callback.py write_to_logger 74]  157 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:10.0157	[raft_python_callback.py write_to_logger 74]  157 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:10.0157	[raft_python_callback.py write_to_logger 74]  157 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:10.0669	[raft_python_callback.py write_to_logger 74]  669 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:10.0670	[raft_python_callback.py write_to_logger 74]  670 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:10.0670	[raft_python_callback.py write_to_logger 74]  670 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:10.0670	[raft_python_callback.py write_to_logger 74]  670 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:10.0670	[raft_python_callback.py write_to_logger 74]  670 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:11.0181	[raft_python_callback.py write_to_logger 74]  181 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:11.0181	[raft_python_callback.py write_to_logger 74]  181 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:11.0181	[raft_python_callback.py write_to_logger 74]  181 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:11.0181	[raft_python_callback.py write_to_logger 74]  181 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:11.0182	[raft_python_callback.py write_to_logger 74]  182 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:11.0693	[raft_python_callback.py write_to_logger 74]  693 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:11.0694	[raft_python_callback.py write_to_logger 74]  694 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:11.0694	[raft_python_callback.py write_to_logger 74]  694 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:11.0694	[raft_python_callback.py write_to_logger 74]  694 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:11.0694	[raft_python_callback.py write_to_logger 74]  694 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:11.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:11.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:12.0205	[raft_python_callback.py write_to_logger 74]  205 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:12.0205	[raft_python_callback.py write_to_logger 74]  205 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:12.0205	[raft_python_callback.py write_to_logger 74]  205 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:12.0205	[raft_python_callback.py write_to_logger 74]  205 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:12.0206	[raft_python_callback.py write_to_logger 74]  206 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:12.0615	[raft_python_callback.py write_to_logger 74]  615 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:12.0615	[raft_python_callback.py write_to_logger 74]  615 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:12.0615	[raft_python_callback.py write_to_logger 74]  615 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:12.0615	[raft_python_callback.py write_to_logger 74]  615 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:12.0616	[raft_python_callback.py write_to_logger 74]  616 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:13.0127	[raft_python_callback.py write_to_logger 74]  127 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:13.0127	[raft_python_callback.py write_to_logger 74]  127 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:13.0128	[raft_python_callback.py write_to_logger 74]  128 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:13.0128	[raft_python_callback.py write_to_logger 74]  128 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:13.0128	[raft_python_callback.py write_to_logger 74]  128 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:13.0639	[raft_python_callback.py write_to_logger 74]  639 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:13.0640	[raft_python_callback.py write_to_logger 74]  640 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:13.0640	[raft_python_callback.py write_to_logger 74]  640 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:13.0640	[raft_python_callback.py write_to_logger 74]  640 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:13.0640	[raft_python_callback.py write_to_logger 74]  640 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:13.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:13.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:14.0151	[raft_python_callback.py write_to_logger 74]  151 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:14.0151	[raft_python_callback.py write_to_logger 74]  151 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:14.0152	[raft_python_callback.py write_to_logger 74]  152 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:14.0152	[raft_python_callback.py write_to_logger 74]  152 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:14.0152	[raft_python_callback.py write_to_logger 74]  152 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:14.0664	[raft_python_callback.py write_to_logger 74]  664 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:14.0664	[raft_python_callback.py write_to_logger 74]  664 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:14.0664	[raft_python_callback.py write_to_logger 74]  664 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:14.0664	[raft_python_callback.py write_to_logger 74]  664 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:14.0664	[raft_python_callback.py write_to_logger 74]  664 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:15.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:15.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:15.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:15.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:15.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:15.0687	[raft_python_callback.py write_to_logger 74]  687 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:15.0688	[raft_python_callback.py write_to_logger 74]  688 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:15.0688	[raft_python_callback.py write_to_logger 74]  688 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:15.0689	[raft_python_callback.py write_to_logger 74]  689 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:15.0689	[raft_python_callback.py write_to_logger 74]  689 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:15.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:15.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:16.0199	[raft_python_callback.py write_to_logger 74]  199 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:16.0200	[raft_python_callback.py write_to_logger 74]  200 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:16.0200	[raft_python_callback.py write_to_logger 74]  200 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:16.0200	[raft_python_callback.py write_to_logger 74]  200 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:16.0200	[raft_python_callback.py write_to_logger 74]  200 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:16.0711	[raft_python_callback.py write_to_logger 74]  711 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:16.0711	[raft_python_callback.py write_to_logger 74]  711 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:16.0711	[raft_python_callback.py write_to_logger 74]  711 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:16.0711	[raft_python_callback.py write_to_logger 74]  711 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:16.0711	[raft_python_callback.py write_to_logger 74]  711 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:17.0120	[raft_python_callback.py write_to_logger 74]  120 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:17.0121	[raft_python_callback.py write_to_logger 74]  121 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:17.0121	[raft_python_callback.py write_to_logger 74]  121 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:17.0121	[raft_python_callback.py write_to_logger 74]  121 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:17.0121	[raft_python_callback.py write_to_logger 74]  121 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:17.0632	[raft_python_callback.py write_to_logger 74]  632 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:17.0633	[raft_python_callback.py write_to_logger 74]  633 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:17.0633	[raft_python_callback.py write_to_logger 74]  633 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:17.0633	[raft_python_callback.py write_to_logger 74]  633 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:17.0633	[raft_python_callback.py write_to_logger 74]  633 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:17.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:17.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:18.0145	[raft_python_callback.py write_to_logger 74]  145 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:18.0145	[raft_python_callback.py write_to_logger 74]  145 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:18.0145	[raft_python_callback.py write_to_logger 74]  145 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:18.0145	[raft_python_callback.py write_to_logger 74]  145 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:18.0145	[raft_python_callback.py write_to_logger 74]  145 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:18.0657	[raft_python_callback.py write_to_logger 74]  657 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:18.0657	[raft_python_callback.py write_to_logger 74]  657 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:18.0657	[raft_python_callback.py write_to_logger 74]  657 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:18.0658	[raft_python_callback.py write_to_logger 74]  658 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:18.0658	[raft_python_callback.py write_to_logger 74]  658 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:19.0169	[raft_python_callback.py write_to_logger 74]  169 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:19.0169	[raft_python_callback.py write_to_logger 74]  169 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:19.0169	[raft_python_callback.py write_to_logger 74]  169 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:19.0169	[raft_python_callback.py write_to_logger 74]  169 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:19.0170	[raft_python_callback.py write_to_logger 74]  170 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:19.0681	[raft_python_callback.py write_to_logger 74]  681 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:19.0681	[raft_python_callback.py write_to_logger 74]  681 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:19.0681	[raft_python_callback.py write_to_logger 74]  681 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:19.0682	[raft_python_callback.py write_to_logger 74]  682 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:19.0682	[raft_python_callback.py write_to_logger 74]  682 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:19.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:19.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:20.0193	[raft_python_callback.py write_to_logger 74]  193 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:20.0193	[raft_python_callback.py write_to_logger 74]  193 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:20.0193	[raft_python_callback.py write_to_logger 74]  193 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:20.0193	[raft_python_callback.py write_to_logger 74]  193 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:20.0193	[raft_python_callback.py write_to_logger 74]  193 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:20.0705	[raft_python_callback.py write_to_logger 74]  705 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:20.0705	[raft_python_callback.py write_to_logger 74]  705 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:20.0706	[raft_python_callback.py write_to_logger 74]  706 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:20.0706	[raft_python_callback.py write_to_logger 74]  706 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:20.0706	[raft_python_callback.py write_to_logger 74]  706 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:21.0114	[raft_python_callback.py write_to_logger 74]  114 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:21.0114	[raft_python_callback.py write_to_logger 74]  114 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:21.0114	[raft_python_callback.py write_to_logger 74]  114 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:21.0114	[raft_python_callback.py write_to_logger 74]  114 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:21.0115	[raft_python_callback.py write_to_logger 74]  115 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:21.0627	[raft_python_callback.py write_to_logger 74]  627 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:21.0627	[raft_python_callback.py write_to_logger 74]  627 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:21.0627	[raft_python_callback.py write_to_logger 74]  627 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:21.0627	[raft_python_callback.py write_to_logger 74]  627 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:21.0628	[raft_python_callback.py write_to_logger 74]  628 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:21.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:21.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:22.0166	[raft_python_callback.py write_to_logger 74]  166 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:22.0167	[raft_python_callback.py write_to_logger 74]  167 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:22.0167	[raft_python_callback.py write_to_logger 74]  167 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:22.0167	[raft_python_callback.py write_to_logger 74]  167 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:22.0167	[raft_python_callback.py write_to_logger 74]  167 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:22.0650	[raft_python_callback.py write_to_logger 74]  650 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:22.0650	[raft_python_callback.py write_to_logger 74]  650 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:22.0650	[raft_python_callback.py write_to_logger 74]  650 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:22.0650	[raft_python_callback.py write_to_logger 74]  650 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:22.0651	[raft_python_callback.py write_to_logger 74]  651 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:23.0674	[raft_python_callback.py write_to_logger 74]  674 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:23.0674	[raft_python_callback.py write_to_logger 74]  674 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:23.0674	[raft_python_callback.py write_to_logger 74]  674 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:23.0674	[raft_python_callback.py write_to_logger 74]  674 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:23.0674	[raft_python_callback.py write_to_logger 74]  674 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:23.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:23.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:24.0187	[raft_python_callback.py write_to_logger 74]  187 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:24.0187	[raft_python_callback.py write_to_logger 74]  187 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:24.0187	[raft_python_callback.py write_to_logger 74]  187 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:24.0187	[raft_python_callback.py write_to_logger 74]  187 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:24.0187	[raft_python_callback.py write_to_logger 74]  187 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:24.0392	[raft_python_callback.py write_to_logger 74]  392 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=180 
-----------------------------

09/03/2018 02:14:24.0393	[raft_python_callback.py write_to_logger 74]  393 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:24.0393	[raft_python_callback.py write_to_logger 74]  393 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=2  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:24.0393	[raft_python_callback.py write_to_logger 74]  393 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:24.0393	[raft_python_callback.py write_to_logger 74]  393 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=2  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:24.0393	[raft_python_callback.py write_to_logger 74]  393 root [DEBUG] 
FILE:./C/Code/follower_handlers.c, FUNCTION:follower_hb_set_log_handler, LINE:10
MESSAGE:follower get set log msg
VARIABLES:  node->msg_data.set_log_hb_msg.cmd=add 
	 node->msg_data.set_log_hb_msg.key=c 
	 node->msg_data.set_log_hb_msg.value=3 
	
-----------------------------

09/03/2018 02:14:24.0393	[raft_python_callback.py write_to_logger 74]  393 root [DEBUG] 
FILE:./C/Code/follower_handlers.c, FUNCTION:follower_hb_set_log_handler, LINE:13
MESSAGE:follower get set log msg
VARIABLES:  sharedRaftData.raft_state.term=11  sharedRaftData.raft_state.last_commit_index=0  node->msg_data.set_log_hb_msg.commit_id=1  node->term=11 
-----------------------------

09/03/2018 02:14:24.0395	[raft_python_callback.py write_to_logger 74]  395 root [CRITI] 
T
09/03/2018 02:14:24.0395	[raft_python_callback.py write_to_logger 74]  395 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=applied_last_idx 
-----------------------------

09/03/2018 02:14:24.0395	[raft_python_callback.py write_to_logger 74]  395 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=1 
-----------------------------

09/03/2018 02:14:24.0398	[raft_python_callback.py write_to_logger 74]  398 root [DEBUG] 
FILE:./C/Code/static_queue.c, FUNCTION:create_new_queue_node_data, LINE:107
MESSAGE:create new msg
VARIABLES:  event=3 
-----------------------------

09/03/2018 02:14:24.0398	[raft_python_callback.py write_to_logger 74]  398 root [DEBUG] 
FILE:./C/Code/follower_handlers.c, FUNCTION:follower_hb_set_log_handler, LINE:37
MESSAGE:follower sending set log res msg
-----------------------------

09/03/2018 02:14:24.0398	[raft_python_callback.py write_to_logger 74]  398 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:send_raft_message, LINE:100
MESSAGE:try to send msg
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:24.0398	[raft_python_callback.py write_to_logger 74]  398 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:send_raft_message, LINE:118
MESSAGE:msg was successfully sent
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:24.0398	[raft_python_callback.py write_to_logger 74]  398 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:24.0398	[raft_python_callback.py write_to_logger 74]  398 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=1  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:24.0398	[raft_python_callback.py write_to_logger 74]  398 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:24.0595	[raft_python_callback.py write_to_logger 74]  595 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:24.0596	[raft_python_callback.py write_to_logger 74]  596 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:24.0596	[raft_python_callback.py write_to_logger 74]  596 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=6  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:24.0596	[raft_python_callback.py write_to_logger 74]  596 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:24.0596	[raft_python_callback.py write_to_logger 74]  596 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=6  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:24.0596	[raft_python_callback.py write_to_logger 74]  596 root [DEBUG] 
FILE:./C/Code/follower_handlers.c, FUNCTION:follower_commit_ok_handler, LINE:91
MESSAGE:follower get commit ok msg
VARIABLES:  sharedRaftData.raft_state.term=11  sharedRaftData.raft_state.last_log_index=1  node->msg_data.commit_ok_msg.last_log_index=1  node->term=11 
-----------------------------

09/03/2018 02:14:24.0596	[raft_python_callback.py write_to_logger 74]  596 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:16
MESSAGE:update DB
VARIABLES:  DB_flag=status  key=commit_idx 
-----------------------------

09/03/2018 02:14:24.0596	[raft_python_callback.py write_to_logger 74]  596 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:update_DB, LINE:17
MESSAGE:update DB
VARIABLES:  value=11 
-----------------------------

09/03/2018 02:14:24.0600	[raft_python_callback.py write_to_logger 74]  600 root [CRITI] 
T
09/03/2018 02:14:25.0210	[raft_python_callback.py write_to_logger 74]  210 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:25.0210	[raft_python_callback.py write_to_logger 74]  210 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:25.0210	[raft_python_callback.py write_to_logger 74]  210 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:25.0210	[raft_python_callback.py write_to_logger 74]  210 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:25.0211	[raft_python_callback.py write_to_logger 74]  211 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:25.0620	[raft_python_callback.py write_to_logger 74]  620 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:25.0620	[raft_python_callback.py write_to_logger 74]  620 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:25.0620	[raft_python_callback.py write_to_logger 74]  620 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:25.0620	[raft_python_callback.py write_to_logger 74]  620 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:25.0620	[raft_python_callback.py write_to_logger 74]  620 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:25.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:25.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:26.0132	[raft_python_callback.py write_to_logger 74]  132 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:26.0132	[raft_python_callback.py write_to_logger 74]  132 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:26.0132	[raft_python_callback.py write_to_logger 74]  132 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:26.0132	[raft_python_callback.py write_to_logger 74]  132 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:26.0133	[raft_python_callback.py write_to_logger 74]  133 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:26.0644	[raft_python_callback.py write_to_logger 74]  644 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:26.0644	[raft_python_callback.py write_to_logger 74]  644 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:26.0644	[raft_python_callback.py write_to_logger 74]  644 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:26.0645	[raft_python_callback.py write_to_logger 74]  645 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:26.0645	[raft_python_callback.py write_to_logger 74]  645 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:27.0156	[raft_python_callback.py write_to_logger 74]  156 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:27.0156	[raft_python_callback.py write_to_logger 74]  156 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:27.0156	[raft_python_callback.py write_to_logger 74]  156 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:27.0157	[raft_python_callback.py write_to_logger 74]  157 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:27.0157	[raft_python_callback.py write_to_logger 74]  157 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:27.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:27.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:28.0180	[raft_python_callback.py write_to_logger 74]  180 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:28.0180	[raft_python_callback.py write_to_logger 74]  180 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:28.0180	[raft_python_callback.py write_to_logger 74]  180 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:28.0180	[raft_python_callback.py write_to_logger 74]  180 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:28.0180	[raft_python_callback.py write_to_logger 74]  180 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:28.0691	[raft_python_callback.py write_to_logger 74]  691 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:28.0692	[raft_python_callback.py write_to_logger 74]  692 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:28.0692	[raft_python_callback.py write_to_logger 74]  692 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:28.0692	[raft_python_callback.py write_to_logger 74]  692 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:28.0692	[raft_python_callback.py write_to_logger 74]  692 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:29.0204	[raft_python_callback.py write_to_logger 74]  204 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:29.0204	[raft_python_callback.py write_to_logger 74]  204 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:29.0204	[raft_python_callback.py write_to_logger 74]  204 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:29.0204	[raft_python_callback.py write_to_logger 74]  204 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:29.0204	[raft_python_callback.py write_to_logger 74]  204 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:29.0613	[raft_python_callback.py write_to_logger 74]  613 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:29.0614	[raft_python_callback.py write_to_logger 74]  614 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:29.0614	[raft_python_callback.py write_to_logger 74]  614 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:29.0614	[raft_python_callback.py write_to_logger 74]  614 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:29.0614	[raft_python_callback.py write_to_logger 74]  614 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:29.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:29.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:30.0126	[raft_python_callback.py write_to_logger 74]  126 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:30.0126	[raft_python_callback.py write_to_logger 74]  126 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:30.0126	[raft_python_callback.py write_to_logger 74]  126 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:30.0126	[raft_python_callback.py write_to_logger 74]  126 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:30.0126	[raft_python_callback.py write_to_logger 74]  126 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:30.0638	[raft_python_callback.py write_to_logger 74]  638 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:30.0638	[raft_python_callback.py write_to_logger 74]  638 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:30.0638	[raft_python_callback.py write_to_logger 74]  638 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:30.0638	[raft_python_callback.py write_to_logger 74]  638 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:30.0638	[raft_python_callback.py write_to_logger 74]  638 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:31.0150	[raft_python_callback.py write_to_logger 74]  150 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:31.0150	[raft_python_callback.py write_to_logger 74]  150 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:31.0150	[raft_python_callback.py write_to_logger 74]  150 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:31.0150	[raft_python_callback.py write_to_logger 74]  150 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:31.0151	[raft_python_callback.py write_to_logger 74]  151 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:31.0661	[raft_python_callback.py write_to_logger 74]  661 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:31.0661	[raft_python_callback.py write_to_logger 74]  661 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:31.0661	[raft_python_callback.py write_to_logger 74]  661 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:31.0661	[raft_python_callback.py write_to_logger 74]  661 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:31.0662	[raft_python_callback.py write_to_logger 74]  662 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:31.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:31.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:32.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:32.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:32.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:32.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:32.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:33.0198	[raft_python_callback.py write_to_logger 74]  198 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:33.0198	[raft_python_callback.py write_to_logger 74]  198 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:33.0198	[raft_python_callback.py write_to_logger 74]  198 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:33.0199	[raft_python_callback.py write_to_logger 74]  199 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:33.0199	[raft_python_callback.py write_to_logger 74]  199 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:33.0710	[raft_python_callback.py write_to_logger 74]  710 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:33.0710	[raft_python_callback.py write_to_logger 74]  710 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:33.0710	[raft_python_callback.py write_to_logger 74]  710 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:33.0710	[raft_python_callback.py write_to_logger 74]  710 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:33.0710	[raft_python_callback.py write_to_logger 74]  710 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:33.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:33.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:34.0118	[raft_python_callback.py write_to_logger 74]  118 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:34.0119	[raft_python_callback.py write_to_logger 74]  119 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:34.0119	[raft_python_callback.py write_to_logger 74]  119 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:34.0119	[raft_python_callback.py write_to_logger 74]  119 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:34.0119	[raft_python_callback.py write_to_logger 74]  119 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:34.0631	[raft_python_callback.py write_to_logger 74]  631 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:34.0631	[raft_python_callback.py write_to_logger 74]  631 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:34.0631	[raft_python_callback.py write_to_logger 74]  631 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:34.0632	[raft_python_callback.py write_to_logger 74]  632 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:34.0632	[raft_python_callback.py write_to_logger 74]  632 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:35.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:35.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:35.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:35.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:35.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:35.0655	[raft_python_callback.py write_to_logger 74]  655 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:35.0656	[raft_python_callback.py write_to_logger 74]  656 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:35.0656	[raft_python_callback.py write_to_logger 74]  656 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:35.0657	[raft_python_callback.py write_to_logger 74]  657 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:35.0657	[raft_python_callback.py write_to_logger 74]  657 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:35.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:35.0928	[raft_python_callback.py write_to_logger 74]  928 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:36.0168	[raft_python_callback.py write_to_logger 74]  168 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:36.0168	[raft_python_callback.py write_to_logger 74]  168 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:36.0168	[raft_python_callback.py write_to_logger 74]  168 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:36.0168	[raft_python_callback.py write_to_logger 74]  168 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:36.0169	[raft_python_callback.py write_to_logger 74]  169 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:36.0679	[raft_python_callback.py write_to_logger 74]  679 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:36.0679	[raft_python_callback.py write_to_logger 74]  679 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:36.0679	[raft_python_callback.py write_to_logger 74]  679 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:36.0680	[raft_python_callback.py write_to_logger 74]  680 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:36.0680	[raft_python_callback.py write_to_logger 74]  680 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:37.0192	[raft_python_callback.py write_to_logger 74]  192 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:37.0192	[raft_python_callback.py write_to_logger 74]  192 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:37.0192	[raft_python_callback.py write_to_logger 74]  192 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:37.0193	[raft_python_callback.py write_to_logger 74]  193 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:37.0193	[raft_python_callback.py write_to_logger 74]  193 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:37.0703	[raft_python_callback.py write_to_logger 74]  703 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:37.0703	[raft_python_callback.py write_to_logger 74]  703 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:37.0703	[raft_python_callback.py write_to_logger 74]  703 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:37.0704	[raft_python_callback.py write_to_logger 74]  704 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:37.0704	[raft_python_callback.py write_to_logger 74]  704 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:37.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:37.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:38.0215	[raft_python_callback.py write_to_logger 74]  215 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:38.0215	[raft_python_callback.py write_to_logger 74]  215 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:38.0215	[raft_python_callback.py write_to_logger 74]  215 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:38.0215	[raft_python_callback.py write_to_logger 74]  215 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:38.0215	[raft_python_callback.py write_to_logger 74]  215 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:38.0625	[raft_python_callback.py write_to_logger 74]  625 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:38.0625	[raft_python_callback.py write_to_logger 74]  625 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:38.0625	[raft_python_callback.py write_to_logger 74]  625 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:38.0625	[raft_python_callback.py write_to_logger 74]  625 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:38.0625	[raft_python_callback.py write_to_logger 74]  625 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:39.0142	[raft_python_callback.py write_to_logger 74]  142 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:39.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:39.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:39.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:39.0143	[raft_python_callback.py write_to_logger 74]  143 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:39.0649	[raft_python_callback.py write_to_logger 74]  649 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:39.0649	[raft_python_callback.py write_to_logger 74]  649 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:39.0650	[raft_python_callback.py write_to_logger 74]  650 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:39.0650	[raft_python_callback.py write_to_logger 74]  650 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:39.0650	[raft_python_callback.py write_to_logger 74]  650 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:39.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:39.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:40.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:40.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:40.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:40.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:40.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:40.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:40.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:40.0686	[raft_python_callback.py write_to_logger 74]  686 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:40.0687	[raft_python_callback.py write_to_logger 74]  687 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:40.0687	[raft_python_callback.py write_to_logger 74]  687 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:41.0184	[raft_python_callback.py write_to_logger 74]  184 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:41.0184	[raft_python_callback.py write_to_logger 74]  184 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:41.0185	[raft_python_callback.py write_to_logger 74]  185 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:41.0186	[raft_python_callback.py write_to_logger 74]  186 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:41.0186	[raft_python_callback.py write_to_logger 74]  186 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:41.0697	[raft_python_callback.py write_to_logger 74]  697 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:41.0697	[raft_python_callback.py write_to_logger 74]  697 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:41.0698	[raft_python_callback.py write_to_logger 74]  698 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:41.0698	[raft_python_callback.py write_to_logger 74]  698 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:41.0698	[raft_python_callback.py write_to_logger 74]  698 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:41.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:41.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:42.0209	[raft_python_callback.py write_to_logger 74]  209 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:42.0209	[raft_python_callback.py write_to_logger 74]  209 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:42.0209	[raft_python_callback.py write_to_logger 74]  209 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:42.0209	[raft_python_callback.py write_to_logger 74]  209 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:42.0210	[raft_python_callback.py write_to_logger 74]  210 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:42.0618	[raft_python_callback.py write_to_logger 74]  618 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:42.0619	[raft_python_callback.py write_to_logger 74]  619 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:42.0619	[raft_python_callback.py write_to_logger 74]  619 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:42.0619	[raft_python_callback.py write_to_logger 74]  619 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:42.0619	[raft_python_callback.py write_to_logger 74]  619 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:43.0131	[raft_python_callback.py write_to_logger 74]  131 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:43.0131	[raft_python_callback.py write_to_logger 74]  131 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:43.0131	[raft_python_callback.py write_to_logger 74]  131 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:43.0131	[raft_python_callback.py write_to_logger 74]  131 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:43.0131	[raft_python_callback.py write_to_logger 74]  131 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:43.0642	[raft_python_callback.py write_to_logger 74]  642 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:43.0642	[raft_python_callback.py write_to_logger 74]  642 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:43.0642	[raft_python_callback.py write_to_logger 74]  642 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:43.0642	[raft_python_callback.py write_to_logger 74]  642 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:43.0642	[raft_python_callback.py write_to_logger 74]  642 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:43.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:43.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:44.0666	[raft_python_callback.py write_to_logger 74]  666 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:44.0666	[raft_python_callback.py write_to_logger 74]  666 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:44.0666	[raft_python_callback.py write_to_logger 74]  666 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:44.0666	[raft_python_callback.py write_to_logger 74]  666 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:44.0667	[raft_python_callback.py write_to_logger 74]  667 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:45.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:45.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:45.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:45.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:45.0179	[raft_python_callback.py write_to_logger 74]  179 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:45.0690	[raft_python_callback.py write_to_logger 74]  690 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:45.0691	[raft_python_callback.py write_to_logger 74]  691 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:45.0691	[raft_python_callback.py write_to_logger 74]  691 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:45.0691	[raft_python_callback.py write_to_logger 74]  691 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:45.0692	[raft_python_callback.py write_to_logger 74]  692 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:45.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:45.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:46.0202	[raft_python_callback.py write_to_logger 74]  202 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:46.0202	[raft_python_callback.py write_to_logger 74]  202 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:46.0202	[raft_python_callback.py write_to_logger 74]  202 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:46.0202	[raft_python_callback.py write_to_logger 74]  202 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:46.0202	[raft_python_callback.py write_to_logger 74]  202 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:46.0715	[raft_python_callback.py write_to_logger 74]  715 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:46.0715	[raft_python_callback.py write_to_logger 74]  715 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:46.0715	[raft_python_callback.py write_to_logger 74]  715 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:46.0715	[raft_python_callback.py write_to_logger 74]  715 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:46.0715	[raft_python_callback.py write_to_logger 74]  715 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:47.0124	[raft_python_callback.py write_to_logger 74]  124 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:47.0124	[raft_python_callback.py write_to_logger 74]  124 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:47.0125	[raft_python_callback.py write_to_logger 74]  125 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:47.0125	[raft_python_callback.py write_to_logger 74]  125 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:47.0125	[raft_python_callback.py write_to_logger 74]  125 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:47.0637	[raft_python_callback.py write_to_logger 74]  637 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:47.0637	[raft_python_callback.py write_to_logger 74]  637 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:47.0637	[raft_python_callback.py write_to_logger 74]  637 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:47.0637	[raft_python_callback.py write_to_logger 74]  637 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:47.0637	[raft_python_callback.py write_to_logger 74]  637 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:47.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:47.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:48.0148	[raft_python_callback.py write_to_logger 74]  148 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:48.0148	[raft_python_callback.py write_to_logger 74]  148 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:48.0148	[raft_python_callback.py write_to_logger 74]  148 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:48.0149	[raft_python_callback.py write_to_logger 74]  149 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:48.0149	[raft_python_callback.py write_to_logger 74]  149 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:49.0172	[raft_python_callback.py write_to_logger 74]  172 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:49.0172	[raft_python_callback.py write_to_logger 74]  172 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:49.0172	[raft_python_callback.py write_to_logger 74]  172 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:49.0172	[raft_python_callback.py write_to_logger 74]  172 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:49.0172	[raft_python_callback.py write_to_logger 74]  172 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:49.0684	[raft_python_callback.py write_to_logger 74]  684 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:49.0684	[raft_python_callback.py write_to_logger 74]  684 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:49.0684	[raft_python_callback.py write_to_logger 74]  684 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:49.0685	[raft_python_callback.py write_to_logger 74]  685 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:49.0685	[raft_python_callback.py write_to_logger 74]  685 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:49.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:49.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:50.0196	[raft_python_callback.py write_to_logger 74]  196 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:50.0196	[raft_python_callback.py write_to_logger 74]  196 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:50.0197	[raft_python_callback.py write_to_logger 74]  197 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:50.0197	[raft_python_callback.py write_to_logger 74]  197 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:50.0197	[raft_python_callback.py write_to_logger 74]  197 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:50.0708	[raft_python_callback.py write_to_logger 74]  708 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:50.0709	[raft_python_callback.py write_to_logger 74]  709 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:50.0709	[raft_python_callback.py write_to_logger 74]  709 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:50.0709	[raft_python_callback.py write_to_logger 74]  709 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:50.0709	[raft_python_callback.py write_to_logger 74]  709 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:51.0118	[raft_python_callback.py write_to_logger 74]  118 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:51.0118	[raft_python_callback.py write_to_logger 74]  118 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:51.0118	[raft_python_callback.py write_to_logger 74]  118 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:51.0118	[raft_python_callback.py write_to_logger 74]  118 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:51.0118	[raft_python_callback.py write_to_logger 74]  118 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:51.0630	[raft_python_callback.py write_to_logger 74]  630 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:51.0630	[raft_python_callback.py write_to_logger 74]  630 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:51.0630	[raft_python_callback.py write_to_logger 74]  630 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:51.0630	[raft_python_callback.py write_to_logger 74]  630 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:51.0631	[raft_python_callback.py write_to_logger 74]  631 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:51.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:51.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:52.0142	[raft_python_callback.py write_to_logger 74]  142 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:52.0142	[raft_python_callback.py write_to_logger 74]  142 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:52.0142	[raft_python_callback.py write_to_logger 74]  142 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:52.0142	[raft_python_callback.py write_to_logger 74]  142 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:52.0142	[raft_python_callback.py write_to_logger 74]  142 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:52.0654	[raft_python_callback.py write_to_logger 74]  654 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:52.0654	[raft_python_callback.py write_to_logger 74]  654 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:52.0654	[raft_python_callback.py write_to_logger 74]  654 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:52.0654	[raft_python_callback.py write_to_logger 74]  654 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:52.0654	[raft_python_callback.py write_to_logger 74]  654 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:53.0678	[raft_python_callback.py write_to_logger 74]  678 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:53.0678	[raft_python_callback.py write_to_logger 74]  678 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:53.0678	[raft_python_callback.py write_to_logger 74]  678 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:53.0678	[raft_python_callback.py write_to_logger 74]  678 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:53.0679	[raft_python_callback.py write_to_logger 74]  679 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:53.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:53.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:54.0189	[raft_python_callback.py write_to_logger 74]  189 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:54.0190	[raft_python_callback.py write_to_logger 74]  190 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:54.0190	[raft_python_callback.py write_to_logger 74]  190 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:54.0190	[raft_python_callback.py write_to_logger 74]  190 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:54.0190	[raft_python_callback.py write_to_logger 74]  190 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:54.0705	[raft_python_callback.py write_to_logger 74]  705 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:54.0705	[raft_python_callback.py write_to_logger 74]  705 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:54.0705	[raft_python_callback.py write_to_logger 74]  705 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:54.0705	[raft_python_callback.py write_to_logger 74]  705 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:54.0705	[raft_python_callback.py write_to_logger 74]  705 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:55.0214	[raft_python_callback.py write_to_logger 74]  214 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:55.0214	[raft_python_callback.py write_to_logger 74]  214 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:55.0214	[raft_python_callback.py write_to_logger 74]  214 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:55.0214	[raft_python_callback.py write_to_logger 74]  214 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:55.0214	[raft_python_callback.py write_to_logger 74]  214 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:55.0623	[raft_python_callback.py write_to_logger 74]  623 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:55.0623	[raft_python_callback.py write_to_logger 74]  623 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:55.0623	[raft_python_callback.py write_to_logger 74]  623 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:55.0623	[raft_python_callback.py write_to_logger 74]  623 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:55.0623	[raft_python_callback.py write_to_logger 74]  623 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:55.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:55.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:56.0135	[raft_python_callback.py write_to_logger 74]  135 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:56.0135	[raft_python_callback.py write_to_logger 74]  135 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:56.0136	[raft_python_callback.py write_to_logger 74]  136 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:56.0136	[raft_python_callback.py write_to_logger 74]  136 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:56.0136	[raft_python_callback.py write_to_logger 74]  136 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:56.0649	[raft_python_callback.py write_to_logger 74]  649 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:56.0649	[raft_python_callback.py write_to_logger 74]  649 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:56.0649	[raft_python_callback.py write_to_logger 74]  649 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:56.0649	[raft_python_callback.py write_to_logger 74]  649 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:56.0649	[raft_python_callback.py write_to_logger 74]  649 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:57.0160	[raft_python_callback.py write_to_logger 74]  160 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:57.0160	[raft_python_callback.py write_to_logger 74]  160 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:57.0160	[raft_python_callback.py write_to_logger 74]  160 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:57.0160	[raft_python_callback.py write_to_logger 74]  160 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:57.0160	[raft_python_callback.py write_to_logger 74]  160 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:57.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:57.0927	[raft_python_callback.py write_to_logger 74]  927 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:58.0183	[raft_python_callback.py write_to_logger 74]  183 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:58.0184	[raft_python_callback.py write_to_logger 74]  184 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:58.0184	[raft_python_callback.py write_to_logger 74]  184 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:58.0184	[raft_python_callback.py write_to_logger 74]  184 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:58.0184	[raft_python_callback.py write_to_logger 74]  184 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:58.0695	[raft_python_callback.py write_to_logger 74]  695 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:58.0696	[raft_python_callback.py write_to_logger 74]  696 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:58.0696	[raft_python_callback.py write_to_logger 74]  696 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:58.0696	[raft_python_callback.py write_to_logger 74]  696 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:58.0696	[raft_python_callback.py write_to_logger 74]  696 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:59.0208	[raft_python_callback.py write_to_logger 74]  208 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:59.0208	[raft_python_callback.py write_to_logger 74]  208 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:59.0208	[raft_python_callback.py write_to_logger 74]  208 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:59.0208	[raft_python_callback.py write_to_logger 74]  208 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:59.0208	[raft_python_callback.py write_to_logger 74]  208 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:59.0616	[raft_python_callback.py write_to_logger 74]  616 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:14:59.0616	[raft_python_callback.py write_to_logger 74]  616 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:14:59.0616	[raft_python_callback.py write_to_logger 74]  616 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:14:59.0617	[raft_python_callback.py write_to_logger 74]  617 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:14:59.0617	[raft_python_callback.py write_to_logger 74]  617 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:59.0930	[raft_python_callback.py write_to_logger 74]  930 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:14:59.0930	[raft_python_callback.py write_to_logger 74]  930 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:15:00.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:15:00.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:15:00.0161	[raft_python_callback.py write_to_logger 74]  161 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:15:00.0162	[raft_python_callback.py write_to_logger 74]  162 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:15:00.0162	[raft_python_callback.py write_to_logger 74]  162 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:15:00.0641	[raft_python_callback.py write_to_logger 74]  641 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:15:00.0641	[raft_python_callback.py write_to_logger 74]  641 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:15:00.0641	[raft_python_callback.py write_to_logger 74]  641 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:15:00.0641	[raft_python_callback.py write_to_logger 74]  641 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:15:00.0641	[raft_python_callback.py write_to_logger 74]  641 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:15:01.0153	[raft_python_callback.py write_to_logger 74]  153 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:15:01.0153	[raft_python_callback.py write_to_logger 74]  153 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:15:01.0154	[raft_python_callback.py write_to_logger 74]  154 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:15:01.0154	[raft_python_callback.py write_to_logger 74]  154 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:15:01.0154	[raft_python_callback.py write_to_logger 74]  154 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:15:01.0665	[raft_python_callback.py write_to_logger 74]  665 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:155
MESSAGE:msg was successfully read
VARIABLES:  message_size=24 
-----------------------------

09/03/2018 02:15:01.0665	[raft_python_callback.py write_to_logger 74]  665 root [DEBUG] 
FILE:./raft_main.c, FUNCTION:run_multicast_listener, LINE:15
MESSAGE:got new message and ids as followed:
VARIABLES:  new_node.message_sent_by=2  sharedRaftData.raft_state.server_id=1 
-----------------------------

09/03/2018 02:15:01.0665	[raft_python_callback.py write_to_logger 74]  665 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:is_relevant_message, LINE:167
MESSAGE:check if msg is relevant
VARIABLES:  sharedRaftData.raft_state.current_state=0  node_message->event=1  is_relevant_rv=1  node_message->term=11 
-----------------------------

09/03/2018 02:15:01.0665	[raft_python_callback.py write_to_logger 74]  665 root [DEBUG] 
FILE:./C/Code/raft_server.c, FUNCTION:get_raft_message, LINE:131
MESSAGE:try to read message
-----------------------------

09/03/2018 02:15:01.0665	[raft_python_callback.py write_to_logger 74]  665 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=1  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:15:01.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/handlers_utils.c, FUNCTION:time_out_hendler, LINE:76
MESSAGE:add new timeout event- now in queue
VARIABLES:  sharedRaftData.raft_state.current_state=0 
-----------------------------

09/03/2018 02:15:01.0926	[raft_python_callback.py write_to_logger 74]  926 root [DEBUG] 
FILE:./C/Code/state_machine.c, FUNCTION:operate_machine_state, LINE:8
MESSAGE:enter state function
VARIABLES:  node->event=0  sharedRaftData.raft_state.current_state=0 
-----------------------------

